{
  "affected": [
    {
      "database_specific": {
        "source": "https://storage.googleapis.com/cve-osv-conversion/osv-output/CVE-2018-11766.json"
      },
      "ranges": [
        {
          "events": [
            {
              "introduced": "cd915e1e8d9d0131462a0b7301586c175728a282"
            },
            {
              "last_affected": "085099c66cf28be31604560c376fa282e69282b8"
            }
          ],
          "repo": "https://github.com/apache/hadoop",
          "type": "GIT"
        }
      ],
      "versions": []
    }
  ],
  "aliases": [
    "GHSA-rqj9-cq6j-958r"
  ],
  "details": "In Apache Hadoop 2.7.4 to 2.7.6, the security fix for CVE-2016-6811 is incomplete. A user who can escalate to yarn user can possibly run arbitrary commands as root user.",
  "id": "CVE-2018-11766",
  "modified": "2024-11-21T03:43:59Z",
  "published": "2018-11-27T14:29:00Z",
  "references": [
    {
      "type": "ADVISORY",
      "url": "http://www.securityfocus.com/bid/106035"
    },
    {
      "type": "WEB",
      "url": "https://lists.apache.org/thread.html/ff37bbbe09d5f03090e2dd2c3dea95de16ef4249e731f19b8959ce4c%40%3Cgeneral.hadoop.apache.org%3E"
    }
  ],
  "schema_version": "1.7.0",
  "severity": [
    {
      "score": "CVSS:3.0/AV:N/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:H",
      "type": "CVSS_V3"
    }
  ]
}